dkg/dkg.go

```diff
@@ -210,6 +214,15 @@ func Run(ctx context.Context, conf Config) (err error) {
 		log.Debug(ctx, "Saved keyshares to disk")
 	}
 
+	if conf.Publish {
+		cl := launchpad.New(conf.LaunchpadAPIAddr)
+		if err = cl.PublishLock(ctx, lock); err != nil {
```

Comment:
see `keymanager.VerifyConnection` we need to ensure BEFORE we start DKG that the API is available to fail fast.

Reply:
It is non blocking on failure so added a verify connection in right before but not sure if it serves much of a purpose?